### ISSUE TITLE
Interpolation in an interval

### ISSUE THREAD

**Issue by ahaverty** [i18next/i18next-intervalPlural-postProcessor#206]
Is it possible to interpolate within an interval?
E.g:
```json
"my_interval": "(1){A {{names[0]}} order is not yet seen};(2){{{names[0]}} and {{names[1]}} orders are not yet seen};(3-inf){{{names[0]}}, {{names[1]}} and other orders are not yet seen};",
```

**Comment by jamuhl** [comment]
i guess that does not yet work...but would be easy to add i guess:

just call https://github.com/i18next/i18next/blob/master/src/Translator.js#L179 on the found value: https://github.com/i18next/i18next-intervalPlural-postProcessor/blob/master/src/index.js#L63

```js
   // ...
   if (newKeys) found = translator.translate(newKeys, newOptions);
} else {
  found = translator.extendTranslation(found, key, { ...options, postProcessorNames: null});
}

return found || value;
```

PR with some tests covering this would be welcome....

**Comment by mbret** [comment]
One solution is to use `alt+0174` character between the first `{` and the second `{`
This inject an invisible character that does not takes any space and therefore virtually accept the regex !

**Comment by jamuhl** [comment]
v3.0.0 **BREAKING** changes regex from `/\((\S*)\).*{((.|\n)*)}/` to `/\((\S*)\).*?\[((.|\n)*)\]/` -> from (condition){value} to (condition)[value] to fix [#206](https://github.com/i18next/i18next-intervalPlural-postProcessor/issues/206) and [#221](https://github.com/i18next/i18next-intervalPlural-postProcessor/issues/221)
